cmd/mr_checkout.go

```diff
@@ -25,7 +25,7 @@ var (
 
 // listCmd represents the list command
 var checkoutCmd = &cobra.Command{
-	Use:   "checkout",
+	Use:   "checkout <id>",
 	Short: "Checkout an open merge request",
 	Long:  ``,
 	Args:  cobra.ExactArgs(1),
```

Comment:
Now that I can see it infront of me, fixing #364 should just be a matter of:
```suggestion
	Args:  cobra.MinimumNArgs(1),
```

Reply:
I was thinking in also add the option the handle the "remote" positional argument.
Not sure how desirable it is.. what do you think?

Reply:
Yeah, that's the kinda sad part with this command, it seems to already have support for it, it's just the argument constraint that messes it up.

```
	rn, mrID, err := parseArgs(args)
		rn, mrID, err := parseArgs(args)
		if err != nil {
		if err != nil {
			log.Fatal(err)
			log.Fatal(err)
		}
		}
		mrs, err := lab.MRList(rn, gitlab.ListProjectMergeRequestsOptions{
		mrs, err := lab.MRList(rn, gitlab.ListProjectMergeRequestsOptions{
			IIDs: []int{int(mrID)},
			IIDs: []int{int(mrID)},
		}, 1)
```